Added option to install simple examples for operators (#160)
stackabletech/integration-tests: README.md

    ./create_test_cluster.py --debug --kind --prometheus --operator trino

## Set up a test kind-cluster with cluster examples deployed

> This will install the `simple-<operator>-cluster.yaml` files in the `example` folder of the specified operators.

Reviewer: ```suggestion
This will install the `simple-<operator>-cluster.yaml` files in the `examples` folder of the specified operators.
```

> Example to create trino, hive and opa operator and deploy **all** the `simple` examples in each operator's example folder.

Reviewer: ```suggestion
This will start the Trino, Hive and OPA operators and deploy **all** the `simple` examples in each operator's `examples` folder:
```

    ./create_test_cluster.py --debug --kind --operator trino hive opa --examples

> Example to create trino, hive and opa operator and deploy **only** the `simple` example in the opa operator.

Reviewer: ```suggestion
This will start the Trino, Hive and OPA operators but **only** deploy the `simple` example from the OPA operator.
```
